test/rpc/test_rpc_misc.py

```diff
@@ -200,6 +201,59 @@ def test_get_events_wrong_blockid_type():
     assert resp["error"]["code"] == PredefinedRpcErrorCode.INVALID_PARAMS.value
 
 
+@pytest.mark.usefixtures("run_devnet_in_background")
+@pytest.mark.parametrize(
+    "run_devnet_in_background",
+    [PREDEPLOY_ACCOUNT_CLI_ARGS],
+    indirect=True,
+)
+def test_get_events_continuation_token():
+    """
+    Test RPC get_events returning continuation token.
+    """
+    deploy_info = deploy(EVENTS_CONTRACT_PATH)
+    for i in range(3):
```

Comment:
Can you extract this 3 to a variable of this function (something like `total_invokes`)? Then use it where applicable.